### PR TITLE
Bartosz/handle errors properly

### PR DESCRIPTION
**Pull request by BartoszJanota**
Make cli more verbose, handle errors properly, don't throw stacktraces.

### CODE REVIEW

File: dataprocxy/dataprocxy.py [hunk 1]
```diff
@@ -71,53 +72,67 @@ def run(self):
...
+            response = request.execute()
+            cluster_name = response['placement']['clusterName'].encode('utf8')
+            return cluster_name
+        except HttpError as error:
+            self.handle_dataproc_http_error(error)
+
+    def handle_dataproc_http_error(self, error):
+        print ('There was a Cloud Dataproc API call error:\n %s, %s' % (error.resp.status, error._get_reason()))
+        if error.resp.status == 404:
+            print ('Perhaps you meant a different region than the one currently set:\n %s' % (self.region))
```
Comment (jrussek) [inline dataprocxy/dataprocxy.py]: I've been thinking about that, the problem is that it really just means that this particular cluster doesn't exist here, which can be a typo the wrong project or many other reasons why dataprocxy cannot find the cluster in this particular place. Maybe giving hints on what to check (is the cluster/job-id correct? did you specify the right project? have you set the correct region?) is more appropriate

**Review by jrussek — COMMENTED** [review]
in general looks good, I'd personally prefer to have a separate commit for the style / whitespace fixes but 🤷‍♂️

File: dataprocxy/dataprocxy.py (hunk 1 above)
Comment (BartoszJanota) [reply dataprocxy/dataprocxy.py]: I agree, let me print out more info when encountering 404 response.

**Review by jrussek — COMMENTED** [review]
👍

**Comment by BartoszJanota** [comment]
@jrussek pls review now, I addressed your comments

**Pull request merged**